functions/create/1.0/function.json

```diff
@@ -49,6 +49,16 @@
       "name": "as",
       "label": "As",
       "info": "The name of the variable you want for the newly created record."
+    },
+    {
+      "meta": {
+        "type": "Boolean",
+        "default": true
```

Comment:
This will fail when releasing to edge. Default value should be added to the JSON schema for the boolean option and implemented in the IDE. I think @justin-betty is gonna do this right?

Reply:
default is already in the json schema: https://raw.githubusercontent.com/bettyblocks/json-schema/master/schemas/actions/function/option/types/boolean.json

Reply:
Yes I will do that

Reply:
@thomas9911 missed that!